apps/web/src/pages/brand/tabs/BrandingForm.tsx

```diff
@@ -82,10 +88,12 @@ export function BrandingForm() {
     setValue('image', path);
   }
 
+  const dropzoneRef = useRef<() => void>(null);
```

Comment:
I created ref for Dropzone, because one of the buttons. must trigger openning file explorer